live/global.ini

```diff
@@ -65729,8 +65729,8 @@ shubin_claimsweep_title_med=Expel Claimjumpers
 shubin_claimsweep_title_rand=Evict Claimjumpers
 shubin_from=Shubin Interstellar
 shuttle_at=Shuttle an
-shuttle_in_transit_to=Shuttle in Bewegung nach
 shuttle_is_at_location=Shuttle-Standort
+shuttle_in_transit_to=Shuttle unterwegs nach
```

Comment:
Richtig das die Reihenfolge getauscht wurde?

Reply:
Ne das war ein Fehler. Ist so wie ich das sehe an der richtigen stelle. Ich hatte eben Mist gemacht ^^

Reply:
Zumindest in meiner lokalen Datei ist es an der richtigen Stelle